cime_config/namelist_definition_drv.xml

```diff
@@ -768,6 +768,31 @@
     </values>
   </entry>
 
+  <entry id="ocn2glc_coupling">
+    <type>logical</type>
+    <category>flds</category>
+    <group>ALLCOMP_attributes</group>
+    <desc>
+      .true. if ocean is sends fields at multiple ocean levels to the land-ice component
+    </desc>
+    <values>
+      <value>.false.</value>
+    </values>
+  </entry>
+
+  <entry id="ocn2glc_levels">
+    <type>char</type>
+    <category>flds</category>
+    <group>ALLCOMP_attributes</group>
+    <desc>
+      if ocean is sends fields at multiple ocean levels to the
```

Comment:
desc needs better wording

Reply:
Why would ocn send to glc at multiple levels?  I don't understand.

Reply:
I'm not sure how to rephrase this. The ocean is sending internal temperature and salinity at 10 levels to the glc. Each level will have a different bathymetry. CISM needs this when it couples the antarctic ice shelf to the ocean. So what would you like me to say? If you can suggest any different wording I'm happy to put it in.

Reply:
"if ocean is sends" maybe just remove the word "is"?